eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -747,6 +747,59 @@ stages:
             targetOS: linux
             targetArchitecture: arm64
 
+        - template: ../jobs/vmr-build.yml
+          parameters:
+            buildName: Mariner_x64_Cross_Alpine
+            isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
+            vmrBranch: ${{ variables.VmrBranch }}
+            architecture: x64
+            pool: ${{ parameters.pool_Linux }}
+            container: ${{ variables.marinerX64AlpineCrossContainer }}
+            crossRootFs: '/crossrootfs/x64'
+            targetOS: linux-musl
+            targetArchitecture: x64
+            extraProperties: /p:TargetRID=linux-musl-x64
+
+        - template: ../jobs/vmr-build.yml
+          parameters:
+            buildName: Mariner_x64_Cross_Alpine_Mono
+            isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
+            vmrBranch: ${{ variables.VmrBranch }}
+            architecture: x64
+            pool: ${{ parameters.pool_Linux }}
+            container: ${{ variables.marinerX64AlpineCrossContainer }}
+            crossRootFs: '/crossrootfs/x64'
+            useMonoRuntime: true
+            targetOS: linux-musl
+            targetArchitecture: x64
+            extraProperties: /p:TargetRID=linux-musl-x64
```

Comment:
Why do we need this?

Reply:
I just noticed that the TargetRID isn't calculated based on TargetOS and TargetArchitecture which seems weird to me but is entirely unrelated to this change.